libraries/tools/kotlin-gradle-plugin/src/common/kotlin/org/jetbrains/kotlin/gradle/plugin/mpp/targetHierarchy/buildKotlinTargetHierarchy.kt

```diff
@@ -117,7 +119,9 @@ private class KotlinTargetHierarchyBuilderImpl(
 
     override fun withAndroidNative() = addTargets { it is KotlinNativeTarget && it.konanTarget.family == Family.ANDROID }
 
-    override fun withJs() = addTargets { it is KotlinJsTargetDsl }
+    /** Don't check for instance of [KotlinJsTargetDsl] or [KotlinWasmTargetDsl] because they are implemented by single target [KotlinJsIrTarget] */
```

Comment:
Please do not use KDoc format for this internal comment. 
Use /* */ or // instead

Reply:
The main reason why I used /** because in that case links in [] are recognised and can be clicked through. But I will change it to just single line comment